scoop-basics/src/main/java/com/example/scoop/basics/ui/layoutsample/layout/NestedView.java

```diff
@@ -0,0 +1,48 @@
+package com.example.scoop.basics.ui.layoutsample.layout;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+import butterknife.Bind;
+import butterknife.ButterKnife;
+import com.example.scoop.basics.R;
+import com.example.scoop.basics.ui.layoutsample.LayoutInjectData;
+import com.example.scoop.basics.ui.layoutsample.module.NestedViewModule;
+import com.lyft.scoop.dagger.DaggerInjector;
+import javax.inject.Inject;
+
+public class NestedView extends LinearLayout{
+
+    @Bind(R.id.nested_text_view)
+    TextView nestedTextView;
+
+    @Inject
+    LayoutInjectData layoutInjectData;
+
+    public NestedView(Context context, AttributeSet attrs) {
+        super(context, attrs);
+        initDagger();
```

Comment:
@itspbj This is a very similar problem we have in codebase right now. You should not extend dagger inside view constructor. This view should already get Scoop with extended dagger injector.